.github/workflows/dependabot-auto-approve.yml

```diff
@@ -13,21 +13,28 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.actor == 'dependabot[bot]' }}
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@1b05615854632b887b69ae1be8cbefe72d3ae423 # v2.6.0
```

Comment:
love this style of comment on the github actions - we'e found it works really well with dependabot!

Reply:
Yeah, I was glad to find that this was possible!